Apps/Src/CLI.c

```diff
@@ -652,28 +654,45 @@ void CLI_Critical(void) {
 	}
 }
 
+/** CLI_OpenWire
+ * Displays open wire status for the BPS system
+ */
+void CLI_OpenWire(void){
+	Voltage_OpenWireSummary();
+}
+
 /** CLI_All
  * Displays all information about BPS modules
- * (voltage, current, temperature, charge, contactor)
+ * (voltage, current, temperature, charge, contactor, open wires)
  */
 void CLI_All(void) {
```

Comment:
The lower-level functions (Voltage, Temperature, etc.) should be pending and posting these mutexes. I know we already discussed at the meeting, so I am just posting this as a reminder.